exercises/standard_library_types/iterators5.rs

```diff
@@ -0,0 +1,113 @@
+// iterators5.rs
+
+// Rustling progress is modelled using a hash map. The name of the exercise is
```

Comment:
```suggestion
// Let's reimplement a simple version of Rustlings exercise progress!
// This will be modelled using a hash map. The name of the exercise is
```